payload/hello_world/main.rs

```diff
@@ -3,9 +3,9 @@
 #![feature(start)]
 // ———————————————————————————————— Guest OS ———————————————————————————————— //
 
-use miralis_abi::{log, setup_binary, success};
+use miralis_abi::{log, setup_binary, setup_payload, success};
```

Comment:
So you need to import both `setup_payload` and `setup_binary` there because `setup_payload` is a macro and expects a macro called `setup_binary` in scope, that's not great...

On second thought I think it is better to stick with `setup_binary` only. Granted the name is not great, but it is easier to maintain a single macro (for instance we don't have a doc comment on `setup_payload` but we do in `setup_macro`). I don't think keeping two distinct macros brings enough benefits on the table, let's keep things simple.